tests/common.py

```diff
@@ -146,25 +148,44 @@ def prepare_reference_nonbonded(params, exclusion_idxs, scales, lambda_plane_idx
     return ref_total_energy
 
 
-def prepare_water_system(x, lambda_plane_idxs, lambda_offset_idxs, p_scale, cutoff):
+def prepare_system_params(x: NDArray, sigma_scale: float = 5.0) -> NDArray:
+    """
+    Prepares random parameters given a set of coordinates. The parameters are adjusted to be the correct
+    order of magnitude.
+
+    Parameters
+    ----------
 
+    x: Numpy array of Coordinates
+
+    sigma_scale: Factor to scale down sigma values by
```

Comment:
Documentation for Return types?

Reply:
Resolved in [fd0fbec](https://github.com/proteneer/timemachine/pull/699/commits/fd0fbecdb11caf2bd64cc9732ef5048d7f8321cb)